specification/analysisservices/resource-manager/readme.md

```diff
@@ -39,6 +39,24 @@ input-file:
 - Microsoft.AnalysisServices/2016-05-16/analysisservices.json

```

Comment:
Please flip order (most recent first). This has no influence on behavior, but makes it consistent with the other RP's configuration in this repo 🙂